src/index.js

```diff
@@ -86,12 +86,13 @@ const getBinaryURL = url => {
   return `${url + separator}postDataBin=y`;
 };
 
-const doJSONPost = (url, { headers = {}, ...otherArgs } = {}) =>
+const doJSONPost = (url, { body, headers = {}, ...otherArgs } = {}) =>
   doPost(getBinaryURL(url), {
     headers: {
       'Content-Type': 'application/json',
       ...headers
     },
+    body: JSON.stringify(body),
```

Comment:
In case one of the applications that's using js-fetch is already stringifying `body` itself, let's allow for that.
```suggestion
    body: typeof body === 'string' ? body : JSON.stringify(body),
```